From 38bdfb4b971a1b59e17c3007ed24ffe44a7fcebd Mon Sep 17 00:00:00 2001
From: Geetha sowjanya <gakula@marvell.com>
Date: Mon, 16 Nov 2020 15:47:33 +0530
Subject: [PATCH 0934/1921] octeontx2-pf: cn10k: Update Marvell license header

This patch updates the license header in all cn10k files.

Change-Id: Ifdad63edf8c821f1c7234ea8069c6394cdf88660
Signed-off-by: Geetha sowjanya <gakula@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/40128
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/rpm.c       | 2 +-
 drivers/net/ethernet/marvell/octeontx2/af/rpm.h       | 6 +-----
 drivers/net/ethernet/marvell/octeontx2/af/rvu_cn10k.c | 5 ++++-
 drivers/net/ethernet/marvell/octeontx2/nic/cn10k.c    | 2 +-
 drivers/net/ethernet/marvell/octeontx2/nic/cn10k.h    | 2 +-
 5 files changed, 8 insertions(+), 9 deletions(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/rpm.c b/drivers/net/ethernet/marvell/octeontx2/af/rpm.c
index 0094c2e907cc..862da2cf7ba9 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/rpm.c
+++ b/drivers/net/ethernet/marvell/octeontx2/af/rpm.c
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: GPL-2.0
 /*  Marvell OcteonTx2 RPM driver
  *
- * Copyright (C) 2020 Marvell International Ltd.
+ * Copyright (C) 2020 Marvell.
  *
  */
 
diff --git a/drivers/net/ethernet/marvell/octeontx2/af/rpm.h b/drivers/net/ethernet/marvell/octeontx2/af/rpm.h
index ce32213a63ba..930909955750 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/rpm.h
+++ b/drivers/net/ethernet/marvell/octeontx2/af/rpm.h
@@ -1,11 +1,7 @@
 /* SPDX-License-Identifier: GPL-2.0 */
 /*  Marvell OcteonTx2 RPM driver
  *
- * Copyright (C) 2020 Marvell International Ltd.
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2 as
- * published by the Free Software Foundation.
+ * Copyright (C) 2020 Marvell.
  */
 
 #ifndef RPM_H
diff --git a/drivers/net/ethernet/marvell/octeontx2/af/rvu_cn10k.c b/drivers/net/ethernet/marvell/octeontx2/af/rvu_cn10k.c
index 0ad8f9702093..81e11bd78d67 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/rvu_cn10k.c
+++ b/drivers/net/ethernet/marvell/octeontx2/af/rvu_cn10k.c
@@ -1,5 +1,8 @@
 // SPDX-License-Identifier: GPL-2.0
-/* Marvell OcteonTx2 RVU Admin Function driver */
+/* Marvell OcteonTx2 RVU Admin Function driver
+ *
+ * Copyright (C) 2020 Marvell.
+ */
 
 #include <linux/bitfield.h>
 #include <linux/pci.h>
diff --git a/drivers/net/ethernet/marvell/octeontx2/nic/cn10k.c b/drivers/net/ethernet/marvell/octeontx2/nic/cn10k.c
index 44cb9b7d0ff5..1b5a292a59b0 100644
--- a/drivers/net/ethernet/marvell/octeontx2/nic/cn10k.c
+++ b/drivers/net/ethernet/marvell/octeontx2/nic/cn10k.c
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: GPL-2.0
 /* Marvell OcteonTx2 RVU Physcial Function ethernet driver
  *
- * Copyright (C) 2018 Marvell International Ltd.
+ * Copyright (C) 2020 Marvell.
  */
 
 #include "cn10k.h"
diff --git a/drivers/net/ethernet/marvell/octeontx2/nic/cn10k.h b/drivers/net/ethernet/marvell/octeontx2/nic/cn10k.h
index ffef7e8f642c..e0bc595cbb78 100644
--- a/drivers/net/ethernet/marvell/octeontx2/nic/cn10k.h
+++ b/drivers/net/ethernet/marvell/octeontx2/nic/cn10k.h
@@ -1,7 +1,7 @@
 /* SPDX-License-Identifier: GPL-2.0
  * Marvell OcteonTx2 RVU Ethernet driver
  *
- * Copyright (C) 2018 Marvell International Ltd.
+ * Copyright (C) 2020 Marvell.
  */
 
 #ifndef CN10K_H
-- 
2.31.1

